db/migrate/20211216171216_create_email_suppressions.rb

```diff
@@ -0,0 +1,9 @@
+class CreateEmailSuppressions < ActiveRecord::Migration[5.2]
+  def change
+    create_table :email_suppressions do |t|
+      t.text :email
+      t.text :suppression_type
+      t.timestamps null: false
+    end
```

Comment:
You should `add_index :email_suppressions, [:email, :suppression_type]` to support your most common query.